### ISSUE TITLE
[Quest] The Ring of Blood

### ISSUE THREAD

**Issue by mariogotze** [wow-mania/Redemption#63]
ID to the first quest of the chain ID#9962
The whole chain can't be completed for 2 or more guys at a time. If we are 2 in group we have to kill each boss twice to complete it for both.

**Comment by ChemicalSerenity** [comment]
To expand (note - entire party was one faction, so not a cross-faction issue):

The problem seems to happen around the Blue Brothers quest in the chain (9967). When I tried it, we killed the brothers but I did not get benefit for the kill despite the yell triggers congratulating me on success. When I abandoned the quest and retook it, two pairs of bosses spawned, all 4 requiring killing before the quest was marked complete.

After that, any additional quest on the chain being taken would be offered to the group, but when anyone in the group clicked "accept", the quest would NOT go in their quest log, and only the original quest taker would get credit.


As it currently sits, the chain is doable by everyone, but every single person needs to kill each boss in turn, which yields 25 or so combats.
